matlab/src/Util.cpp

```diff
@@ -47,19 +47,15 @@ namespace
     mxArray* cellArrayToString(mxArray* cellArray)
     {
         mxArray* stringArray;
-        mxArray* params[1];
-        params[0] = cellArray;
-        mexCallMATLAB(1, &stringArray, 1, params, "string");
+        mexCallMATLAB(1, &stringArray, 1, &cellArray, "string");
         return stringArray;
     }
 
-    // This function converts the string array input argument and returns a cell array of char arrays.
+    // This function converts the string array input argument and returns a cell array of char.
     mxArray* stringToCellArray(mxArray* stringArray)
     {
         mxArray* cellArray;
-        mxArray* params[1];
```

Comment:
We don't need a params with a single argument: we can just take the reference of the single param.